### PR TITLE
feature: Added the sponsors section.

### PR DESCRIPTION
**Pull request by ishimwe07**
I created a section of sponsors, which contains companies,
It has like 2 parts, first is a heading and descriptive paragraph and the other part contains 6 companies(name and logo for each).

### CODE REVIEW

File: src/sections/maibornwolff.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            
+            <div class="flex flex-col gap-44 align-bottom font-railwayRegular text-base">
+                <div>
+                    <div class="flex">
+                        <img class="w-16 h-16" src="/images/image 1.webp" alt="BMW logo">
+                        
+                        <img class="w-44 h-11" src="/images/stihl.svg" alt="STIHL logo">
+                        
+                        <img class="w-32 h-16" src="/images/Miele_Logo_M_Red_sRGB.svg" alt="Miele logo">                  
+                    </div>
```
Comment (divinecharlotte) [inline src/sections/maibornwolff.html]: - [x] kindly consider making sure that these images have the same color as the one provided in the Figma to stick to the given design

File: src/sections/maibornwolff.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                            <span class="font-railwayBold">internationaler IT Dienstleister</span> digitalisieren 
+                            lassen
+                        </p>
+                    </div>
+                </div>
+    
+                <div>
+                    <div class="flex">
+                        <img class="w-40 h-14"  src="/images/google.svg" alt="google logo">
+                        <img class="w-48 h-16" src="/images/african-union.svg" alt="African Union image">
```
Comment (divinecharlotte) [inline src/sections/maibornwolff.html]: - [x] kindly consider making sure that these images have the same color as the one provided in the Figma to stick to the given design

File: src/sections/maibornwolff.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                    <div class="w-2/5 mt-3">
+                        <p class="font-railwayRegular">
+                            haben bereits die Power des <span class="font-railwayBold">Sub-Saharan Gamesstudio</span> genutzt, um geniale Spiele entwickeln zu lassen.
+                        </p>
+                    </div>
+                </div>
+    
+                <div class="w-2/5 pt-4">
+                    <p class="font-railwayRegular text-base">
+                        Viele Talente setzen auf Ruandas intensivstes
```
Comment (divinecharlotte) [inline src/sections/maibornwolff.html]: - [x] kindly consider breaking this paragraph as it is in the Figma design to give the expected design

File: src/sections/maibornwolff.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                        
+                        <img class="w-44 h-11" src="/images/stihl.svg" alt="STIHL logo">
+                        
+                        <img class="w-32 h-16" src="/images/Miele_Logo_M_Red_sRGB.svg" alt="Miele logo">                  
+                    </div>
+                    <div class="w-2/5 mt-3">
+                        <p>
+                            haben Produkte und Prozesse vom   
+                            <span class="font-railwayBold">internationaler IT Dienstleister</span> digitalisieren 
+                            lassen
```
Comment (divinecharlotte) [inline src/sections/maibornwolff.html]: - [x]  kindly consider breaking this paragraph(adding `br` tags) as it is in the Figma design to give the expected design

File: src/sections/maibornwolff.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+                </div>
+    
+                <div>
+                    <div class="flex">
+                        <img class="w-40 h-14"  src="/images/google.svg" alt="google logo">
+                        <img class="w-48 h-16" src="/images/african-union.svg" alt="African Union image">
+                    </div>
+                    <div class="w-2/5 mt-3">
+                        <p class="font-railwayRegular">
+                            haben bereits die Power des <span class="font-railwayBold">Sub-Saharan Gamesstudio</span> genutzt, um geniale Spiele entwickeln zu lassen.
```
Comment (divinecharlotte) [inline src/sections/maibornwolff.html]: - [x]  kindly consider breaking this paragraph(adding br tags) as it is in the Figma design to give the expected design

File: src/sections/maibornwolff.html [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+</head>
+<body>
+    <section class="pb-40">
+        <div class="mt-48 mb-20 ml-36 w-1/3">
+            <h1 class="font-railwayBold text-3xl">MaibornWolff x The Gym x Maliyo</h1>
+            <p class="font-railwayRegular text-lg py-4">
+                Für das Training werden die Erfahrung und das 
+                Wissen über erfolgreiche Softwareprojekte 
+                kombiniert mit dem standortspezifischen ...
+            </p>
```
Comment (divinecharlotte) [inline src/sections/maibornwolff.html]: - [x]  kindly consider breaking this paragraph(adding br tags) as it is in the Figma design to give the expected design

File: src/sections/maibornwolff.html [hunk 7]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ojemba</title>
+</head>
+<body>
+    <section class="pb-40">
```
Comment (divinecharlotte) [inline src/sections/maibornwolff.html:2]: - [x] kindly consider giving this section the same oblique borders on top and bottom to make your design an exact duplicate of the given one in Figma 
hint 😀  you can download the background from figma and directly use it to minimize work

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @ishimwe07   , 

Good job so far!
You still need to work on some issues to go to the part, but you are almost there!


 




 

**Check the comments under the review._**

Cheers, and Happy coding!👏👏👏

File: src/sections/maibornwolff.html [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+                        
+                        <img class="w-14 h-8 sm:w-24 sm:h-9 lg:w-44 lg:h-14" src="/images/Stihl_Logo 1-modified.webp" alt="STIHL logo">
+                        
+                        <img class="w-16 h-10 sm:w-28 sm:h-10 lg:w-32 lg:h-16" src="/images/Miele_Logo_M_Red_sRGB-modified.webp" alt="Miele logo">                  
+                    </div>
+                    <div class="lg:mt-3">
+                        <p>
+                            haben Produkte und Prozesse vom <br class="hidden sm:block"><span class="font-railwayBold">internationaler IT Dienstleister</span> digitalisieren 
+                            <br class="hidden sm:block">lassen
+                        </p>
```
Comment (NelHim) [inline src/sections/maibornwolff.html:40]: Make the indentation cleaner

**Review by NelHim — CHANGES_REQUESTED** [review]

File: src/sections/maibornwolff.html (hunk 8 above)
Comment (ishimwe07) [reply src/sections/maibornwolff.html:40]: Done.

File: src/sections/maibornwolff.html (hunk 5 above)
Comment (ishimwe07) [reply src/sections/maibornwolff.html]: Done.
You can check.

File: src/sections/maibornwolff.html (hunk 3 above)
Comment (ishimwe07) [reply src/sections/maibornwolff.html]: Done

**Review by divinecharlotte — APPROVED** [review]

**Review by NelHim — APPROVED** [review]

**Pull request merged**